patch_topology.sh

```
@@ -0,0 +1,5 @@
+sed -i 's/let transform_count = if spec.config.transform.is_some() {/let transform_count = usize::from(spec.config.transform.is_some());/g' crates/logfwd-runtime/src/pipeline/topology.rs
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)